### PR TITLE
Entity+database

### PR DESCRIPTION
**Pull request by pro100bodia**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <artifactId>NutritionTracking</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>war</packaging>
+
+    <name>NutritionTracking Maven Webapp</name>
+    <url>http://localhost:8080</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.7</maven.compiler.source>
```
Comment (valeriiafatova) [inline pom.xml:17]: why not 1.8?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <maven.compiler.target>1.7</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
```
Comment (valeriiafatova) [inline pom.xml:25]: better move to properties in file

File: src/main/java/com/bod/entity/Client.java [hunk 3]
```diff
@@ -0,0 +1,119 @@
+package com.bod.entity;
+
+import java.time.LocalDate;
+
+public class Client {
+    private int id;
+    private String name;
+    private String password;
+
```
Comment (valeriiafatova) [inline src/main/java/com/bod/entity/Client.java:9]: remove epmty spaces

File: src/main/java/com/bod/entity/Client.java [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    private DailyRation dailyRation;
+
+    private DeflectionHistory deflectionHistory;
+    private LocalDate birthDate;
+    private GenderEnum gender;
+    private double height;
+    private double weight;
+    private LifeStyleEnum lifeStyle;
```
Comment (valeriiafatova) [inline src/main/java/com/bod/entity/Client.java:17]: better not to use 'Enum" in enum classes naming

File: src/main/java/com/bod/entity/Client.java [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    private NutritiveValue norm;
```
Comment (valeriiafatova) [inline src/main/java/com/bod/entity/Client.java:18]: better to rename to full and sensible name

File: src/main/java/com/bod/entity/DailyRation.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
+package com.bod.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DailyRation {
+    private List<Food> foodList;
```
Comment (valeriiafatova) [inline src/main/java/com/bod/entity/DailyRation.java:7]: if it is entity, i would consider an id

File: src/main/java/com/bod/entity/DeflectionHistory.java [hunk 7]
```diff
@@ -0,0 +1,44 @@
+package com.bod.entity;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class DeflectionHistory {
+    private NutritiveValue value;
```
Comment (valeriiafatova) [inline src/main/java/com/bod/entity/DeflectionHistory.java:7]: if it is and Entity, i would consider an id

File: src/main/java/com/bod/entity/Food.java [hunk 8]
```diff
@@ -0,0 +1,94 @@
+package com.bod.entity;
+
+import java.util.Objects;
+
+public class Food {
+    private String name;
```
Comment (valeriiafatova) [inline src/main/java/com/bod/entity/Food.java:6]: if it is an entity i would consider an id

File: pom.xml (hunk 2 above)
Comment (pro100bodia) [reply pom.xml:25]: all dependencies vertions are moved to properties in the last commit

File: src/main/java/com/bod/entity/Client.java (hunk 3 above)
Comment (pro100bodia) [reply src/main/java/com/bod/entity/Client.java:9]: spaces between fields declaration are removed

File: src/main/java/com/bod/entity/Client.java (hunk 4 above)
Comment (pro100bodia) [reply src/main/java/com/bod/entity/Client.java:17]: "Enum" prefixes are removed from enum names.

File: src/main/java/com/bod/entity/Client.java (hunk 5 above)
Comment (pro100bodia) [reply src/main/java/com/bod/entity/Client.java:18]: Norm is a set of calories, protein, fats and carbohydrates recomended for the client. NutritiveValue is a "datatype" to store all that three numbers. What name do you prefer?

File: src/main/java/com/bod/entity/DailyRation.java (hunk 6 above)
Comment (pro100bodia) [reply src/main/java/com/bod/entity/DailyRation.java:7]: Id and id getter are add.

File: src/main/java/com/bod/entity/DeflectionHistory.java (hunk 7 above)
Comment (pro100bodia) [reply src/main/java/com/bod/entity/DeflectionHistory.java:7]: Id and id getter are add.

File: src/main/java/com/bod/entity/Food.java (hunk 8 above)
Comment (pro100bodia) [reply src/main/java/com/bod/entity/Food.java:6]: Id and id getter are add.

**Pull request merged**